src/colors.js

```diff
@@ -113,25 +113,38 @@ export const LEGO_COLORS = [
   [137, 135, 136],
 ];
 
-export const closestMatch = (rgb) => {
+
+export const closestMatch = (rgb, colorOptions) => {
```

Comment:
Looks like these changes broke the unit test in `colors.spec.js`. Can you fix that?

Reply:
Will do. Thanks for the suggestions. Also your updated interface looks really neat!